app/src/main/java/com/kylecombes/micrepeater/ScreenSlidePageFragment.java

```diff
@@ -1,19 +1,20 @@
 package com.kylecombes.micrepeater;
 
+import android.content.Context;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.Button;
-import android.widget.TextView;
+import android.widget.CompoundButton;
+import android.widget.Switch;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.core.content.ContextCompat;
 import androidx.fragment.app.Fragment;
 
 public class ScreenSlidePageFragment extends Fragment {
-    private static final String ARG_COUNT = "param1";
+    private static final String ARG_COUNT = "COUNTER";
     private Integer counter;
```

Comment:
Perhaps `PAGE_INDEX` and `mPageIndex`? "Counter" is pretty ambiguous.

Reply:
Fixed